### ISSUE TITLE
implementation of the loss function from paper

### ISSUE THREAD

**Issue by Irrelifant** [Mikoto10032/AutomaticWeightedLoss#10]
Hey everyone, 

first of all, thanks for your implementation 
the following formula from the paper  "Auxiliary Tasks in Multi-task Learning" 
![image](https://user-images.githubusercontent.com/56792395/159053816-1f7b79bc-f66f-4dbd-8300-a40bb6e69697.png)

was implemented by: 

            loss_sum += 0.5 / (self.params[i] ** 2) * loss + torch.log(1 + self.params[i] ** 2)

but imo this is not exactly true, since 0.5 / sigma^2 is not the same as 1/(2*sigma^2) am I right or do i oversee something here?

Thanks for feedback

**Comment by ress4859** [comment]
1/(2*sigma^2)= 1/2 *(1/sigma^2)=0.5*(1/sigma^2) = 0.5/(sigma^2)
